src/components/ourAdvantages/OurAdvantages.js

```diff
@@ -1,39 +1,107 @@
 import Image from 'next/image';
 
-const advantages = [
-  {
-    iconSrc: '',
-    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-  },
-  {
-    iconSrc: '',
-    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-  },
-  {
-    iconSrc: '',
-    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-  },
-  {
-    iconSrc: '',
-    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
-  },
-];
-
 const OurAdvantages = () => {
   return (
-    <section className='grid h-auto min-h-[350px] justify-items-center bg-Green-600 py-[15px] sm:grid-cols-2 lg:flex lg:items-center lg:justify-center xl:gap-[40px]'>
-      {advantages.map((advantage, idx) => (
-        <div
-          key={idx}
-          className='grid h-[180px] max-w-[260px] content-center justify-items-center gap-[15px] lg:w-[250px] xl:w-[260px]'
-        >
-          <div className='h-[50px] w-[50px] bg-Green-300'>
-            <Image src={advantage.iconSrc} alt='icon' width={50} height={50} />
+    <>
+      <section className='grid grid-cols-1 gap-5 md:grid-cols-3'>
+        <div className='border  border-gray-200 p-7'>
+          <div className='block-advantage group transition duration-700'>
```

Comment:
I think it would be better to move this block of code into a separate component to avoid code duplication
```
<div className='block-advantage group transition duration-700'>
     <Image src='/advatageImages/planning.svg' alt='planning' width={70} height={70} />
     <h2 className='transition duration-300 group-hover:text-Yellow-500'>
        Project Planning
     </h2>
     <p className='text-gray-400'>
         Lorem ipsum dolor sit amet, consectetur adipisicing elit. Architecto deleniti dolore
         est illum iusto, laboriosam provident quasi quod sed sunt.
     </p>
     <div className=''>
        <button className='font-bold transition duration-300 group-hover:text-Yellow-500'>
            Learn more
        </button>
      </div>
</div>
```